app/assets/stylesheets/practices.scss

```diff
@@ -1,3 +1,50 @@
-// Place all the styles related to the Practices controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+h3 {
+  margin: 10px;
+  font-weight: 900;
+}
+
+.profile {
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.